김은지/[LV2] 전력망을 둘로 나누기.js

```diff
@@ -0,0 +1,41 @@
+function getDividedNetwork(wires, removeWireId) {
+  const newWires = wires.slice();
+  newWires.splice(removeWireId, 1);
+  return newWires;
+}
+
+function getGraph(n, linked) {
+  const graph = Array.from({ length: n + 1 }, () => []);
+  linked.forEach(([src, dst]) => {
+    graph[src].push(dst);
+    graph[dst].push(src);
+  });
+  return graph;
+}
+
+function dfs(n, graph) {
+  const visited = Array.from({ length: n + 1 }, () => false);
+  visited[1] = true;
+  const stack = [1];
+
+  let cnt = 0;
+  while (stack.length > 0) {
+    const src = stack.pop();
+    cnt++;
+
+    for (const dst of graph[src]) {
+      if (visited[dst]) continue;
+      stack.push(dst);
+      visited[dst] = true;
+    }
+  }
+  return cnt;
+}
+
+function solution(n, wires) {
+  let minDiff = Number.POSITIVE_INFINITY;
+  for (let i = 0; i < n; i++) {
```

Comment:
엇.. 아무 생각없이 범위 n으로 했는데 이부분 원래 n - 1이 되어야 합니다.

```suggestion
  for (let i = 0; i < n - 1; i++) {
```

wires 조건이 n-1개라서 많이 벗어나진 않았지만.. 의도는 wires.length 였습니다! 제외할 wire를 모두 순회하는 게 목적이기 때문에..! 수정해올릴게요!